apps/asap-server/test/controllers/users.test.ts

```diff
@@ -32,86 +30,52 @@ describe('Users controller', () => {
   describe('Fetch', () => {
     describe('with mock-server', () => {
       test('Should fetch the users from squidex graphql', async () => {
-        const result = await usersMockGraphql.fetch({});
+        const result = await usersMockGraphqlServer.fetch({});
 
         expect(result).toMatchObject(getListUserResponse());
       });
     });
 
     describe('with intercepted http layer', () => {
```

Comment:
just so you know the reason i split the mock-server and 'intercepted http layer' in other test files was because of nock and having to add identity and nock expectations in beforeAll and afterEach, if `Fetch` method is tested without nock then you might not need this split - checkout  https://github.com/yldio/asap-hub/blob/93ac40b96a40f4a0ea366b17b1800f1661016e31/apps/asap-server/test/controllers/research-outputs.test.ts#L32 as an example

Reply:
I've removed the describe('with mock-server' and the describe('with intercepted http layer'. It wasn't clear to me, I thought they were to distinguish between mocking the server and the client.